### PR TITLE
Create Calculator.js and add Calculator component

### PR DESCRIPTION
**Pull request by danalvrz**
- Deleted  all the boilerplate from CRA.
- Created a directory called /components.
- Created a  Calculator.js file.
- Developed the UI of the calculator.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>Mathemagicians</h1>
-        <p>coming soon...</p>
-        <p>This is gonna be awesome!</p>
-      </header>
+      <body className="App-body">
+        <Calculator />
+      </body>
     </div>
   );
```
Comment (DeliceLydia) [inline src/App.js]: - [x] App component should be class based

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @danalvrz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! major highlights for your project:
- Followed git worflow
- Descriptive commits messages


## Required Changes ♻️
_Check the comments under the review._

- [x] I suggest that in the `public` folder you can keep just `index,html`
mm

- [x] In the `src` folder I suggest that you keep files that you are using only



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (danalvrz) [reply src/App.js]: Thanks for the comments! I made all the suggested changes

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi, @danalvrz 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**